parlai/tasks/jericho_world/constants.py

```diff
@@ -13,10 +15,22 @@ class GraphMutations(Enum):
     ADD = 2
 
 
+try:
+    stopwords.words('english')
+except LookupError:
+    nltk.download('stopwords')
+
+
+# The set of words in the description of graph that we skip while checking
+# for overlap between knowledge grapj vertices and the description.
+GRAPH_VERT_SKIP_TOKEN = set(stopwords.words('english')) - {'you'}
```

Comment:
Nit - it's useful to include reasonings for this kind of decision when external context drove the decision. Here the choice to leave out "you" is deliberate, but the note just describes what these tokens are used for.

Reply:
Added extra explanation.